webapp/views.py

```diff
@@ -70,70 +69,61 @@ def browse(request):
                         subsubgroup["docs"] = sorted([cache for cache in browse_cache if cache["Type"] == subsubgroup["tag"]], key=lambda x: (x["Year"], x["DisplayName"]) if ("Year" in x) else (0, x["DisplayName"]))
     browse_cache = {cache["document_id"]: cache for cache in browse_cache}
 
-    return render(request, "browse.html", {"browse_groups": browse_list, "cache_data": browse_cache})
+    return flask.render_template(
+        "browse.html",
+        {"browse_groups": browse_list, "cache_data": browse_cache})
 
-@csrf_exempt
-def create(request):
-    print(request)
```

Comment:
Deleting `print`s such as this one is a bit out of scope but they got a bit much…